README.md

```diff
@@ -21,22 +19,14 @@
  You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
 
-
-
-#################
-  INSTRUCTIONS
-#################
+## Instructions
 
 OpenShot Video Editor 2.0 supports Linux, Mac, and Windows. But due to 
```

Comment:
Hmm, maybe update all of these "2.0" references? OpenShot hasn't been version 2.0 for years.

I figure they could either say "OpenShot 2" or "OpenShot 2.x" (if the intent is simply to distinguish it from OpenShot 1.x), or else it should be "OpenShot 2.4" at the moment, and we'll have to try and remember to update it each time OpenShot moves to a new minor-version series.

Reply:
Yeah I was thinking about saying 2.x on it previously. I will make this change.